orttraining/orttraining/python/training/torchdynamo/register_backend.py

```diff
@@ -12,40 +12,18 @@
 
 
 class AotOrt(AotAutogradStrategy):
+    """Implement compiler interface to plug ORT into TorchDynam.
+
+    Under the hood, AotOrt.compile is called inside functorch. See aot_function
+    and aot_module in aot_autograd.py in PyTorch repo for more details. Basically,
+    AotOrt.compile is mapped to forward graph compiler, fw_compile, and backward
+    graph compiler, bw_compile, in aot_autograd.py.
+    """
+
     def __init__(self, graph_module: torch.fx.GraphModule, example_inputs):
         super().__init__(graph_module, example_inputs)
 
         self.ort = OrtBackend()
-        self.populate_aten2aten_decomps()
-
-    def populate_aten2aten_decomps(self):
```

Comment:
ort_backend.py already has a decomposition table when calling `make_fx`, so we don't need this one.